README.md

```diff
@@ -168,6 +168,85 @@ batch.patchItemOperation(testDoc.getId(), cosmosPatchOperations);
 TransactionalBatchResponse batchResponse = container.executeTransactionalBatch(batch).block();
 ~~~~
 
+### NodeJS:
+
+The npm package can be found at [NPM package: com.azure » @azure/cosmos » 3.14.1 (npmjs.com)](https://www.npmjs.com/package/@azure/cosmos/v/3.14.1)
+
+
+#### Patching an item with a single patch operation
+
+```
+const patchSource = itemDefList[1];
```

Comment:
not sure if this line makes sense to have since user doesn't know what itemDefList is

also, not sure if it would be best to use another naming convention for the variables being used (testItem seems to be used above in the file) to be clearer as well